<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>312</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="textField"/>
        </item>
        <item>
         <widget class="QPushButton" name="open_file">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="process">
          <property name="text">
           <string>Process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabwidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="input">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>171</width>
          <height>168</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="selectinputgame">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Select Game</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_3">
           <property name="text">
            <string>GTA III</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_vc">
           <property name="text">
            <string>GTA VC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_sa">
           <property name="text">
            <string>GTA SA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_4">
           <property name="text">
            <string>GTA IV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_5">
           <property name="text">
            <string>GTA V</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_game_bully">
           <property name="text">
            <string>Bully Scholarship Edition﻿</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>101</width>
          <height>113</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="inputype">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="input_type_binary">
           <property name="text">
            <string>Binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="input_type_text">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>171</width>
          <height>168</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="selectoutputgame">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Select Game</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_3">
           <property name="text">
            <string>GTA III</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_vc">
           <property name="text">
            <string>GTA VC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_sa">
           <property name="text">
            <string>GTA SA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_4">
           <property name="text">
            <string>GTA IV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_5">
           <property name="text">
            <string>GTA V</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selec_output_game_bully">
           <property name="text">
            <string>Bully Scholarship Edition﻿</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>101</width>
          <height>113</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="outputtype">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="output_type_binary">
           <property name="text">
            <string>Binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="output_type_text">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="others">
       <attribute name="title">
        <string>Others</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>9</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="strings">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>Strings</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="strings_gta4">
           <property name="text">
            <string>GTA IV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="strings_bully">
           <property name="text">
            <string>Bully</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="None">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>105</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="filetype">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>File Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="file_type_ide">
           <property name="text">
            <string>Ide</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="file_type_ipl">
           <property name="text">
            <string>Ipl</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wpl">
           <property name="text">
            <string>Wpl</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>301</width>
          <height>56</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Postion Mover</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>346</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textField</tabstop>
  <tabstop>open_file</tabstop>
  <tabstop>process</tabstop>
  <tabstop>selec_game_3</tabstop>
  <tabstop>selec_game_vc</tabstop>
  <tabstop>selec_game_sa</tabstop>
  <tabstop>selec_game_4</tabstop>
  <tabstop>selec_game_5</tabstop>
  <tabstop>selec_output_game_3</tabstop>
  <tabstop>selec_output_game_vc</tabstop>
  <tabstop>selec_output_game_sa</tabstop>
  <tabstop>selec_output_game_5</tabstop>
  <tabstop>input_type_binary</tabstop>
  <tabstop>input_type_text</tabstop>
  <tabstop>output_type_binary</tabstop>
  <tabstop>output_type_text</tabstop>
  <tabstop>file_type_ide</tabstop>
  <tabstop>file_type_ipl</tabstop>
  <tabstop>strings_gta4</tabstop>
  <tabstop>strings_bully</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
